test-builder/javascript.ts

```diff
@@ -24,25 +24,17 @@ const stripAttrName = (name, featureName) =>
     .replace(/__(\w+)__/g, "$1");
 
 const shouldIgnoreAttr = (featureName: string, attrName: string) => {
-  const prototypeString = `${featureName}.prototype`;
-  if (attrName === prototypeString) {
-    // Skip the prototype itself
-    return true;
-  }
+  const ignoreList = [
+    "String.prototype.trimLeft()",
+    "String.prototype.trimRight()",
+  ];
 
-  if (
-    featureName === "Atomics" &&
-    ["WaiterRecord", "WaiterListRecords"].includes(attrName)
-  ) {
-    // The spec defines WaiterRecord/WaiterListRecords as if they were members of Atomics
```

Comment:
Why are these lines being removed?

Reply:
I logged all `attrName` that are passed into `shouldIgnoreAttr` and WaiterRecord/WaiterListRecords are never in there in the first place so I think the check is not needed. Maybe I'm misunderstanding this check?

Reply:
It's possible that the spec has since fixed this issue, so we no longer need this ignore in place.  I'm not seeing these present in the generated tests now, so I'm going to assume that's the case.